tests/end2end/test_partition.py

```diff
@@ -22,18 +22,19 @@
 from __future__ import absolute_import, print_function
 
 import warnings
+
 import pytest
 from requests.packages import urllib3
 
 import zhmcclient
 # pylint: disable=line-too-long,unused-import
-from zhmcclient.testutils import hmc_definition, hmc_session  # noqa: F401, E501
 from zhmcclient.testutils import dpm_mode_cpcs  # noqa: F401, E501
-# pylint: enable=line-too-long,unused-import
-
+from zhmcclient.testutils import hmc_definition, hmc_session  # noqa: F401, E501
 from .utils import skip_warn, pick_test_resources, TEST_PREFIX, \
     standard_partition_props, runtest_find_list, runtest_get_properties
 
+# pylint: enable=line-too-long,unused-import
```

Comment:
Why did you move the pylint line? It puts the import from `.utils` into the pylint section where unused import is disabled. That is only necessary for imports that provide pylint fixtures or their dependent fixtures (such as dpm_mode_cpcs, hmc_definition or hmc_session).

If the import from `.utils` has unused imports, they should be removed instead.

Reply:
okay, reverted it back